scripts/manage-version-in-files.mjs

```diff
@@ -60,8 +60,8 @@ function readFileContent(path) {
 
 function getCurrentVersion() {
   const json = readFileContent('./package.json');
-  const pkg = JSON.parse(json);
-  return pkg.version;
+  const package_ = JSON.parse(json);
```

Comment:
**suggestion:** rename into `packageJson` for clarity (the `_` postfix is weird).